pkg/manifestgen/install/manifests.go

```diff
@@ -73,7 +73,7 @@ func generate(base string, options Options) error {
 		// traffic from going through http proxy. Without fully specified
 		// domain they need to mention `notifications-controller` explicitly in
 		// `no_proxy` variable after debugging http proxy logs.
-		options.EventsAddr = fmt.Sprintf("http://%s.%s.svc.%s/", options.NotificationController, options.Namespace, options.ClusterDomain)
+		options.EventsAddr = fmt.Sprintf("http://%s.%s.svc.%s./", options.NotificationController, options.Namespace, options.ClusterDomain)
```

Comment:
Wouldn't it be better to make the ending `.` part of the `ClusterDomain` option? In this way, people can still control _not_ adding a dot using this option.

Reply:
That's breaking change to SC.

Reply:
How is that breaking to SC? The controller is configured using the full domain as `--storage-adv-addr` string.

Reply:
`ClusterDomain` is used by SC, if we remove the dot and we add it to the default value, then we'll break Flux for all those that have set a custom domain, e.g. GKE private.

Reply:
I would be for adding the dot in `options.ClusterDomain`. And here a validation check that adds one if there isn't one already - to avoid double dots.

Reply:
The problem is with people having configured a custom domain, assuming a dot will be added. Which would no longer be added if we change it, which may cause issues (but likely only results in an additional DNS lookup).